exercises/space-age/.meta/generator/space_age_cases.rb

```diff
@@ -0,0 +1,13 @@
+require 'generator/exercise_case'
+
+class SpaceAgeCase < Generator::ExerciseCase
+
+  SECONDS_PER_YEAR_ON_EARTH = 31557600
+
+  def workload
+    indent_lines(["age = SpaceAge.new(#{seconds.underscore})",
+      "    assert_in_delta #{expected}, age.on_#{planet.downcase}, DELTA"
```

Comment:
The spaces between `"` and `assert` are not required when you're using `indent_lines`.